apisix/plugins/consumer-restriction.lua

```diff
@@ -128,7 +128,13 @@ function _M.access(conf, ctx)
     local method = ngx.req.get_method()
 
     if not value then
-        return 401, { message = "Missing authentication or identity verification."}
+        local err_msg
+        if conf.type == "consumer_name" then
+            err_msg = "Missing authentication information in request."
+        else
+            err_msg = "Please ensure " .. conf.type .. " is present in request"
+        end
```

Comment:
Can the two message be unifed into this form?
"The request is rejected becase the " .. conf.type .. " is missing"

Reply:
The request be rejected when `config.type` is `consumer_name` maybe cause by that the user authentication info not match any `consumer`, but not missing configuration of `consumer`. So, prompt user configuration is missing maybe not very suitable.

Reply:
But the msg "Please ensure service_id is present in request" is not clear too, because the `service_id` is not configured in the request.

Reply:
How about "The request is rejected, please check the "  .. conf.type .. " for this request"

Reply:
I think it's ok, thank you for your patient suggestion.